cmd/frontend/internal/httpapi/internal.go

```
@@ -223,6 +223,7 @@ func serveSearchConfiguration(db dbutil.DB) func(http.ResponseWriter, *http.Requ
 		repos, loadReposErr := database.Repos(db).List(ctx, database.ReposListOptions{
 			Names: repoNames,
 			IDs:   indexedIDs,
+			UseOr: true,
```

thought: should we rather return a 400 error if both are specified and make it XOR?

---

Updated the title/description and code to do that instead.